src/hooks/useValidation/useValidation.js

```diff
@@ -103,7 +111,50 @@ const useValidation = (context = {}) => {
     return issues;
   }, [context.action, context.marcType]);
 
-  const runBackEndValidation = useCallback(async (marcRecords) => {
+  // if the length of a subfield of field 008 is shorter, then add backslashes,
+  // if longer, then cut off the extra characters.
+  const fillIn008FieldBlanks = useCallback((marcRecords) => {

```

Comment:
Let's move this out to `prepareForSubmit` in Edit/Create/Derive wrappers. This hook should assume that the provided data is what is going to be sent to BE to save, so it should validate this data without changes

Reply:
This needs to be done only for BE validation. `marcRecords` for BE should contain the correct length of field 008 subfields to avoid 400 error, but FE shouldn't, in order to catch errors and display inline messages.

If we put this functionality in `prepareForSubmit` for BE and FE, we will get the correct length of subfields and will not see inline errors.